### DISCUSSION TITLE
Any idea what errors codes mean in a go executable? 7,52

### DISCUSSION THREAD

**Discussion by working-name** [bdd/runitor#164] [General]
Hi there,

Trying to figure out what's going on with the error codes spit out by runitor v1.3.0-beta.2:

Exit Code: 52(Unknown error), then immediately after the normal status 1 from bash.
![image](https://github.com/bdd/runitor/assets/16505052/621eae0f-a466-48e3-bc7d-39a30368a1fe)

This is the kind of non-descript error message going on when trying to run curl. I could turn curl verbose to investigate, of course. Just wondering if you already have an idea.
```py
Task failed to run properly. 
Here's the error message: The command #(I added newlines for readability)
"'/home/user/cron/php-based/runitor-v1.3.0-beta.2-linux-amd64' 
'-api-url' 'http://172.17.0.1:19999/ping/' '-ping-key' '' '-slug' 
'slug-of-task-here' '-create' '--' 
'curl' '-s' 'http://172.17.0.1:3000/api/...'" failed.

Exit Code: 52(Unknown error)

Working directory: /home/user

Output:
================


Error Output:
================
```

#### More examples:
```py
Exit Code: 7(Unknown error)
[runitor] exit status 7
Exit Code: 52(Unknown error)
...
```

**Comment by bdd** [comment]
As a command runner runitor exits with the exit status of the command it ran. In this example above curl is exiting with code 52. If you drop `-s` from the curl flags I would expect it to complain to stderr that it received no response from the server. The nondescriptness is from curl's invocation, not particularly from runitor.

From https://curl.se/libcurl/c/libcurl-errors.html
```
CURLE_GOT_NOTHING (52)

Nothing was returned from the server, and under the circumstances, getting nothing is considered an error. 
```

**Reply by working-name** [reply]
That did not even cross my mind, but great point. Thank you!
